PublicationsRouter/v2/api/MatchingConfig.md

```diff
@@ -0,0 +1,42 @@
+# Matching Config Structure
+Document contains structure of matching configuration JSON for both submission and retrieval. Used in POST/GET requests respectively against endpoint `/config`.
+
+## JSON Submitted Structure
+The structure of a JSON submission for matching configuration must match the following format: 
```

Comment:
`This JSON structure must be used for submitting a new matching configuration using POST.`